src/adapter/breakpointPredictor.ts

```diff
@@ -153,13 +155,14 @@ class DirectoryScanner {
           source: metadata.sourceUrl,
           line: b.line,
           column: b.column || 1,
+          bias: SourceMapConsumer.LEAST_UPPER_BOUND,
         });
         if (entry.line === null) {
           continue;
         }
 
         const { lineNumber, columnNumber } = uiToRawOffset(
-          { lineNumber: entry.line || 1, columnNumber: entry.column || 1 },
+          { lineNumber: entry.line || 1, columnNumber: entry.column ? entry.column + 1 : 1 },
```

Comment:
This and the bias adjustment above are changes we applied in the 'runtime' breakpoints earlier that I forgot to add over here